packages/freetype.rb

```diff
@@ -3,40 +3,56 @@
 class Freetype < Package
   description 'FreeType is a freely available software library to render fonts.'
   homepage 'https://www.freetype.org/'
-  version '2.11.0'
+  version '2.11.1'
   license 'FTL or GPL-2+'
   compatibility 'all'
-  source_url 'https://download.savannah.gnu.org/releases/freetype/freetype-2.11.0.tar.xz'
-  source_sha256 '8bee39bd3968c4804b70614a0a3ad597299ad0e824bc8aad5ce8aaf48067bde7'
+  source_url 'https://gitlab.freedesktop.org/freetype/freetype.git'
+  git_hashtag "VER-#{version.gsub('.', '-')}"
```

Comment:
`.tr` is better (and a *little* bit faster) in single word global substitution:
```ruby
git_hashtag "VER-#{version.tr('.', '-')}"
```

Reply:
There are a bunch of packages which have that. Feel free to use sed to change that in multiple packages after we get this merged. ;)